buildtools/stylelint/stylelint.js

```diff
@@ -0,0 +1,45 @@
+/* eslint no-console: 0 no-undef: 0 */
+const checkStyle = require('stylelint-checkstyle-formatter');
+const fs = require('fs');
+const path = require('path');
+const stylelint = require('stylelint');
+
+const STYLELINT_OUT_FILE = 'build2/OSW-stylelint-checkstyle.xml';
+const SASS_LINT_FILES = [
+  'assets/sass/**/*.scss',
+  '!assets/sass/widgets/*.scss',
+  '!assets/sass/common/**/*.scss'
+];
+
+const checkStyleEnabled = process.argv.includes('checkstyle');
+const options = {
+  configFile: path.join(__dirname, '../../.stylelintrc.json'),
+  files: SASS_LINT_FILES,
+  formatter: checkStyleEnabled ? checkStyle : 'string'
+};
+
+console.warn(__dirname);
```

Comment:
IIRC the success/failure output contains directory name information. To avoid the current directory name being logged every-time this is task run, we're probably OK removing it and rely explicitly on the output.